web-app/packages/lib/src/modules/layout/components/SideBarTemplate.vue

```diff
@@ -63,20 +63,20 @@ import { useLayoutStore } from '@/modules/layout/store'
 
 const route = useRoute()
 const layoutStore = useLayoutStore()
-layoutStore.init()
-
-const currentPage = computed(() => route.name)
 
 const initialSidebarItems = computed<SideBarItemModel[]>(() => {
   return [
     {
-      active: currentPage.value === 'dashboard',
+      active: route.matched.some((item) => item.name === 'dashboard'),
       title: 'Dashboard',
       to: '/dashboard',
       icon: 'ti ti-home'
     },
     {
-      active: currentPage.value === 'projects',
+      // TODO: hardcoded path names add to enum
```

Comment:
please resolve

Reply:
Implemented enums based on modules. When adding new view to the modules/project/views directory, it's recommended to create corresponding route names for this view. These route names can be used for routing purposes within the library or in other applications.